migrations/1594206146121-set-school-maintenance-dates-2020.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
```

Comment:
Codacy found an issue: ['mongoose' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5827997)